environment/environment.yaml

```diff
@@ -11,6 +11,7 @@ dependencies:
   - networkx=1.11
   - xgboost=0.6a2
   - lightgbm=2.0.4
+  - seaborn=0.8.1
```

Comment:
btw, could you please check that all required modules are also present at setup.py? (everything that is not related to classifiers, e.g. pyyaml)

Reply:
one more thing 
- make colormap the same for different plots, and 
- allow user to pass params to plotting functions like a) colormap, b) number of bins, c) fig_size